honeybee_energy/hvac/heatcool/_base.py

```diff
@@ -74,30 +71,98 @@ def to_ideal_air_equivalent(self):
     def properties(self):
         """Get properties for extensions."""
         return self._properties
-
+    
     @classmethod
     def from_dict(cls, data):
-        new_obj = super(_HeatCoolBase, cls).from_dict(data)
+        """Create a HVAC object from a dictionary.
```

Comment:
I changed this method so that it matches the other hvac (allir / doas) for consistency